backend/app/models/action_log.py

```
@@ -0,0 +1,18 @@
+class ActionLogComment(Base):
```

Ruff (0.12.2)

Undefined name `Base`

(F821)